packages/manager/apps/hpc-vmware-managed-vcd/src/utils/getRefetchConditions.ts

```diff
@@ -0,0 +1,12 @@
+import IVcdDatacentre from '@/types/vcd-datacenter.interface';
+import IVcdOrganization from '@/types/vcd-organization.interface';
+
+export type UpdatableResource = IVcdOrganization | IVcdDatacentre;
+
+const targetSpecKey = 'configure-target-spec';
+
+export const isUpdatingTargetSpec = (resource: UpdatableResource) =>
+  !!resource?.currentTasks?.some((task) => task.type === targetSpecKey);
+
+export const hasResourceUpdatingTargetSpec = (resources: UpdatableResource[]) =>
+  !!resources?.some((resource) => isUpdatingTargetSpec(resource));
```

Comment:
`!!` should not be necessary as `some` already returns `boolean`

```suggestion
  resources?.some((resource) => isUpdatingTargetSpec(resource));
```